packages/react/src/components/filterBox/FilterBox.tsx

```diff
@@ -82,12 +88,25 @@ export class FilterBox extends Component<IFilterBoxProps, any> {
 
     static defaultProps: Partial<IFilterBoxProps> = {
         isAutoFocus: false,
+        disabled: false,
+    };
+
+    private handleIcons = (inputValue: string) => {
+        if (inputValue.length !== 0) {
+            this.filterInput.nextElementSibling.setAttribute('class', ' flex filter-box-icon hidden'); // hide filter icon
+            this.filterInput.nextElementSibling.nextElementSibling.classList.remove('hidden'); // show clear icon
```

Comment:
Can you add in the classList like you do for the remove?
```suggestion
            this.filterInput.nextElementSibling.classList.add('hidden'); // hide filter icon
            this.filterInput.nextElementSibling.nextElementSibling.classList.remove('hidden'); // show clear icon
```